scripts/jenkins/jobs-obs/openstack-rpm-packaging-update.yaml

```diff
@@ -32,3 +32,12 @@
           osc service dr
           osc addremove
           osc commit -m "Automatically updated rpm-packaging-openstack via Jenkins"
+
+          # also create links for new packages
+          echo "creating missing links ..."
+          for x in *-SLE_12_SP1.spec; do
+              pkgname=(${x//-SLE_12_SP1.spec/ })
```

Comment:
Why is this done in a subshell? Wouldn't `pkgname=${x//-SLE_12_SP1.spec/}` do the same, and be more readable?